### PR TITLE
Changes to Pin mapping for POC and initialization

### PR DESCRIPTION
**Pull request by mfruge9214**
I dont want to ruin the stuff you guys worked on, so the only changes I think are important are the ones to bsp, motor_control, and main

### CODE REVIEW

File: src/motor_control.c [hunk 1]
```diff
@@ -64,65 +67,198 @@ void prv_Motor(void *pvParameters) {
...
+    PWMPulseWidthSet(y_motor.PWM_Base, Y_PWM_OUT, 0);
+
+    /* Enable Interrupts */
+    PWMGenIntRegister(y_motor.PWM_Base, y_motor.PWM_Block, PWM0IntHandler);
+
+    /* Enable the generator block to start timer */
+    PWMGenEnable(y_motor.PWM_Base, y_motor.PWM_Block);
 }
 
+/* Commented for test */
```
Comment (jackdski) [inline src/motor_control.c]: use a #ifdef instead of commenting it out

File: src/motor_control.c [hunk 2]
```diff
@@ -138,32 +274,213 @@ void motor_change_pwm_duty_cycle(Motor_t motor, uint8_t duty_cycle) {
...
+//    z_motor.M1.pin    =    Z_M1_PIN;
+//
+//    z_motor.DIR.base  =    Z_DIR_PORT;
+//    z_motor.DIR.pin   =    Z_DIR_PIN;
+//
+//    z_motor.ENABLE.base =  Z_ENABLE_PORT;
+//    z_motor.ENABLE.pin  =  Z_ENABLE_PIN;
+//
+//    z_motor.NSLEEP.base =  Z_NSLEEP_PORT;
+//    z_motor.NSLEEP.pin  =  Z_NSLEEP_PIN;
```
Comment (jackdski) [inline src/motor_control.c]: Use a #ifdef here too

File: include/motor_control.h [hunk 3]
```diff
@@ -48,31 +48,46 @@ typedef struct {
...
     uint32_t        PWM_Channel;
     uint32_t        PWM_Pin_Map;
 } Motor_t;
 
+///* Motor structs, defined in motor_control.c */
+//extern Motor_t x_motor;
+//extern Motor_t y_motor;
+//extern Motor_t z_motor;
+//extern Motor_t ex_motor;
+
```
Comment (jackdski) [inline include/motor_control.h]: Delete these

**Review by jackdski — CHANGES_REQUESTED** [review]

File: include/motor_control.h (hunk 3 above)
Comment (mfruge9214) [reply include/motor_control.h]: Changes made

File: src/motor_control.c (hunk 2 above)
Comment (mfruge9214) [reply src/motor_control.c]: Changes made

File: src/motor_control.c (hunk 1 above)
Comment (mfruge9214) [reply src/motor_control.c]: Changes made

File: include/bsp.h [hunk 4]
```diff
@@ -18,7 +18,73 @@
 /*
  * TODO: Change these to real pins and ports
  */
+#define POC
 
+#ifdef POC
```
Comment (AveryAnderson) [inline include/bsp.h:23]: Some pins undefined in the else case, and may need to be defined post POC. For example  X_M0_PORT is undefined

File: main.c [hunk 5]
```diff
@@ -5,7 +5,7 @@
  */
 
 /*  I N C L U D E S   */
-#include <motor_control.h>
+//#include <motor_control.h>
```
Comment (AveryAnderson) [inline main.c]: Commented include can be removed

**Review by AveryAnderson — APPROVED** [review]

File: main.c (hunk 5 above)
Comment (mfruge9214) [reply main.c]: Done

**Review by jackdski — APPROVED** [review]

**Pull request closed without merging**